bundles/ch.elexis.icpc/src/ch/elexis/icpc/views/EpisodesView.java

```diff
@@ -199,7 +198,7 @@ public void run(){
 		
 		editEpisodeAction = new Action("Problem bearbeiten") {
 			{
-				setToolTipText("Titel des Problems ändern");
+				setToolTipText("Problem bearbeiten");
 				setImageDescriptor(Images.IMG_EDIT.getImageDescriptor());
```

Comment:
Fehlt hier nicht eine Übersetzung?

Reply:
As the ICPC contains at least 250 non externalized strings, I propose that Txomin shall do this work later and that Marco just merges the text change as requested by Matej